lib/gutenberg/gutenberg-editor-sidebar-component.js

```diff
@@ -97,6 +97,21 @@ export default class GutenbergEditorSidebarComponent extends AsyncBaseContainer
 		}
 	}
 
+	async setCommentsPreference( { allow = true } = {} ) {
```

Comment:
Nicely done! I couldn't succeed to click on the checkbox with given id and class selector